relay-ourlogs/src/ourlog.rs

```diff
@@ -29,6 +30,10 @@ pub fn otel_to_sentry_log(otel_log: OtelLog) -> OurLog {
 
     let mut attribute_data = Object::new();
 
+    // We ignore the passed observed time since Relay always acts as the collector in Sentry.
+    // We may change this in the future with forwarding Relays.
+    let observed_time_unix_nano = Utc::now().timestamp_nanos_opt().unwrap_or(0) as u64;
```

Comment:
What is the precision you need on this, generally we'd use `UnixTimestamp::now()` here, which has an `as_secs()`. This limits your precision to per second, but that seems like it should be good enough for the received/observed timestamp?